deploy/cloudwash_job.yaml

```diff
@@ -28,9 +28,10 @@ objects:
                 command:
                   - swach
                 args:
-                  - "-d"
                   - "azure"
-                  - "--all"
+                  - "--nics"
+                  - "--vms"
+                  - "--discs"
```

Comment:
I assume we don't need to cleanup azure "images" if there are any,
then this PR is ready to merge.
Thanks!

Reply:
The `--images` will be a follow up.